vllm/model_executor/layers/attention.py

```diff
@@ -311,6 +311,7 @@ def __init__(
         base: int = 10000,
         num_kv_heads: Optional[int] = None,
         is_neox_style: bool = True,
+        is_glm_style: bool = False,
```

Comment:
Let's use a single style variable:
```suggestion
        style: str = "neox",
```